codechecker_common/plist_parser.py

```diff
@@ -330,7 +329,8 @@ def skip_report_from_plist(plist_file, skip_handler):
     with io.open(plist_file, 'r+') as plist:
         new_plist_content = remove_report_from_plist(plist,
                                                      skip_handler)
-        new_plist_content = new_plist_content.decode("utf8")
-        plist.seek(0)
-        plist.write(new_plist_content)
-        plist.truncate()
+        if new_plist_content:
```

Comment:
`remove_report_from_plist` will always return a string, the modified or the original plist.
This section will be always executed why the `if new_plist_content`?

Reply:
The `plist_content` was undefined in this function:
https://github.com/Ericsson/codechecker/blob/68a1d54245e6eaba3c0ee6b7a27ca916bf0baa50/codechecker_common/plist_parser.py#L285-L322
To solve this we will return with None in case of error so this is the reason why we check the return value.

Reply:
In that case please update the docstring that None will be returned if the remove failed for some reason.